testing/unit/rpc/test_state.py

```diff
@@ -0,0 +1,109 @@
+import pytest
+from aiohttp_json_rpc.exceptions import RpcInvalidParamsError
+
+from remme.rpc_api.state import fetch_state
+from remme.shared.exceptions import KeyNotFound
+from testing.utils._async import (
+    raise_async_error,
+    return_async_value,
+)
+
+
+@pytest.mark.asyncio
+async def test_fetch_state(mocker, request_):
+    """
+    Case: fetch state.
+    Expect: fetches a particular leaf from the current state.
```

Comment:
`particular leaf from the current state is fetched`.

Reply:
@dmytrostriletskyi, done.